test/envoye2e/inventory.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2020 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package envoye2e
+
+import (
+	env "istio.io/proxy/test/envoye2e/env"
+)
+
+var ProxyE2ETests *env.TestInventory
+
+func init() {
+	ProxyE2ETests = &env.TestInventory{
+		Tests: []string{
```

Comment:
This is nice. We could probably parallellize sub-tests as well. I wonder if this could be auto-generated later with `go list`

Reply:
Yeah, seems like a good code gen target. Added a todo for it.